contracts/axelar-interchain-token-service/src/test.rs

```diff
@@ -16,12 +18,8 @@ fn setup_env<'a>() -> (Env, Address, InterchainTokenServiceClient<'a>) {
     (env, contract_id, client)
 }
 
-fn initialize_its(
-    _env: &Env, 
-    client: &InterchainTokenServiceClient,
-    owner: Address,
-) {
-    client.initialize(&owner);
+fn initialize_its(_env: &Env, client: &InterchainTokenServiceClient, owner: Address) {
```

Comment:
```suggestion
fn initialize(_env: &Env, client: &InterchainTokenServiceClient, owner: Address) {
```
test method can be called `initialize` since there's no name collision concern